store/prefixstore_test.go

```diff
@@ -0,0 +1,71 @@
+package store
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/tendermint/iavl"
+	dbm "github.com/tendermint/tmlibs/db"
+)
+
+func testPrefixStore(t *testing.T, prefixStore KVStore) {
```

Comment:
Can we test the actual prefixing (have a prefix store and a non-prefix store, test that keys are prefixed correctly)?